src/pages/_HomePage.scss

```diff
@@ -93,12 +93,15 @@
     }
   }
 
+  .btn-find {
+    background: linear-gradient(308deg, #ff92d3 0%, #46698d 100%);
```

Comment:
I am really sorry Ming, but I think that this gradient is horrible.
In my opinion it is super flashy and the color scheme is not in line with the rest of the website.

Reply:
I just tried this new color scheme on top of current green. And obeyed current Yellow to Green oppostion. To make another opposition. Otherwise, I won't do this.